website/src/examples/carto.js

```diff
@@ -38,7 +38,7 @@ class CartoSQLDemo extends Component {
     week: {displayName: 'Week', type: 'select', options: Object.keys(WEEKS), value: 'Week 1'},
   };
 
-  static mapStyle = 'https://basemaps.cartocdn.com/gl/dark-matter-gl-style/style.json';
+  static mapStyle = 'https://basemaps.cartocdn.com/gl/dark-matter-nolabels-gl-style/style.json';
```

Comment:
- Is it intentional to drop the labels?
- Use `MAPBOX_STYLES` constant like the other examples?

Reply:
Yes, it's

Reply:
I've moved to a constant: 49615c127

Reply:
You should import it from constants/defaults.js

Reply:
oops, you're right. fixed 52036bb

Reply:
@Josmorsot FYI you don't have to reply to every comment. You can just click on the "Resolve conversation" button.